src/command.js

```diff
@@ -32,7 +32,8 @@ export class Command {
             extensions: this.extensions,
             ini: this.ini,
             dependencies: this.dependencies,
-            ignore_platform_reqs_8: this.ignore_platform_reqs_8,
+            ignore_platform_reqs_8: this.ignore_php_platform_requirement,
```

Comment:
When running a job on PHP 7.4, it is absolutely fine to provide `ignore_platform_reqs8` as `false` or `true` depending on the PHP 7.4 configuration.

The action itself will only apply this to the composer command if the PHP version used in the action is PHP 8.0 and thus, we are good to go.

Reply:
This is good - it will ensure that the CI container doesn't need immediate changes, and we can update it _after_ this is already released.

Reply:
> This is good - it will ensure that the CI container doesn't need immediate changes, and we can update it after this is already released.
Yup, that was the idea. This change is fully BC compatible.